src/locales/pl-pl.ts

```diff
@@ -20,6 +20,13 @@ const plPL: Locale = {
     no: () => "Nie",
     yes: () => "Tak",
   },
+  characterCount: {
+    youHave: () => "Masz",
+    tooMany: () => "zbyt wiele",
+    character: () => "znak",
+    characters: () => "znaki",
+    remaining: () => "pozostałe",
+  },
```

Comment:
I'm afraid that won't work for polish language (and might not work for others as well).

I think it might be better to create a template which will give users more flexibilty:


```suggestion
tooManyCharacters: (count) => count === 1 ? `Masz o ${count} znak za dużo` : `Masz o ${count} znaków za dużo`;
charactersLeft: (count) => count === 1 ? `Masz ${count} pozostały znak` : `Masz ${count} pozostałych znaków`;
```

That will also reduce the logic for gluing those words inside of the CharacterCount component

Reply:
I forgot that polish is one of the hardest languages in the world and it won't be that simple, sorry for that :). The template idea I proposed is still valid but for the polish translation it will slightly more complex because we need to use 3 words, not just 2, `znak`, `znaki`, `znaków`.

Here's a method for that:

```
function polishPlural(singularNominativ, pluralNominativ, pluralGenitive, value) {
    if (value === 1) {
        return singularNominativ;
    } else if (value % 10 >= 2 && value % 10 <= 4 && (value % 100 < 10 || value % 100 >= 20)) {
        return pluralNominativ;
    } else {
        return pluralGenitive;
    }
}
```

And then use this method like this:

```
tooManyCharacters: (count) => `Masz o ${count} ${polishPlurals("znak", "znaki", "znaków", count) za dużo`;


charactersLeft: (count) => `Masz ${count} ${polishPlurals("pozostały", "pozostałe", "pozostałych", count) ${polishPlurals("znak", "znaki", "znaków", count)`;

```